komga-webui/src/locales/en.json

```diff
@@ -180,7 +180,7 @@
   "common": {
     "all_libraries": "All Libraries",
     "books": "Books",
-    "books_n": "No book | 1 book | {count} books",
+    "books_n": "0 books | 1 book | {count} books",
```

Comment:
I saw your other link on the use of numericals, but i don't think it applies to 0/none.

Reply:
Neither do I, but the exception is for enumerating.